test/framework/src/main/java/org/opensearch/test/OpenSearchTestCase.java

```diff
@@ -211,7 +211,12 @@
     "LuceneFixedGap",
     "LuceneVarGapFixedInterval",
     "LuceneVarGapDocFreqInterval",
-    "Lucene50" })
+    "Lucene50",
+    "Lucene90",
+    "Lucene94",
+    "Lucene90",
+    "Lucene95",
+    "Lucene99" })
```

Comment:
Huh... interesting that this list wasn't really updated since the Lucene 5 days.

In theory, I think we could clean up some of the old ones, since they might not exist anymore.

Reply:
I was pretty surprised when I saw this as well since I believe we want to use the AssertingCodec 100% of the time. It's probably worth taking a closer look at which codecs are actually still in use but maybe in the context of a separate issue.

Reply:
Created a separate issue to review the test codecs:
- https://github.com/opensearch-project/OpenSearch/issues/12767